platform/x11/os_x11.cpp

```diff
@@ -255,41 +255,13 @@ void OS_X11::initialize(const VideoMode &p_desired, int p_video_driver, int p_au
 
 		visual_server = memnew(VisualServerWrapMT(visual_server, get_render_thread_mode() == RENDER_SEPARATE_THREAD));
 	}
-
-	// borderless fullscreen window mode
-	if (current_videomode.fullscreen) {
```

Comment:
Where did all that code go? Is it no longer relevant?

Reply:
The same code exists inside the set_window_fullscreen function and I call that instead, minus some legacy alternative code for openbox, which isn't needed anymore as far as I can tell, and wasn't called anyway when you explicitly called set_window_fullscreen in gdscript.